deploy/firesim

```diff
@@ -118,16 +118,12 @@ def buildafi(globalbuildconf):
             run("uname -a")
 
     auto_create_bucket(globalbuildconf.s3_bucketname)
-
-    client = boto3.client('sns')
-
-    aws_resource_names_dict = aws_resource_names()
-    snsname = aws_resource_names_dict['snsname']
-
-    # this will either create the topic, if it doesn't exist, or just get the arn
-    client.create_topic(
-        Name=snsname
-    )
+    #check to see email notifications can be subscribed
+    try:
+        create_topic() 
+    except :
```

Comment:
Also, I think it would be better to only have the `client.create_topic` call inside of our new function in the `try` block. It will more precisely catch what we're trying to catch and also handle the permissions errors in other different uses of the new function that we may not think about today.

Reply:
> Also, I think it would be better to only have the `client.create_topic` call inside of our new function in the `try` block.

I didn't get this. We have to anyway get the 'snsname' right to execute 'client.create_topic' ?